src/components/Buttons/docs/RadioCheckbox.md

```diff
@@ -0,0 +1,53 @@
+This component outputs a radio button or a checkbox.
+
+The programming logic behind producing these buttons is the same. The difference is in the props `btnType`. It defaults to radio if undefined.
+
+You can define whether the button starts in the pressed on unpressed state through the `isChecked` props.
```

Comment:
Small typo - `prop` instead of `props`. We should also mention that is can also be controlled past first mount via that prop.